dev/ci/nix/unicoq.nix

```diff
@@ -0,0 +1,14 @@
+{ stdenv, fetchzip, coq }:
+
+stdenv.mkDerivation {
+  name = "coq${coq.coq-version}-unicoq-0.0-git";
+  src = fetchzip {
+    url = "https://github.com/vbgl/unicoq/archive/8b33e37700e92bfd404bf8bf9fe03f1be8928d97.tar.gz";
```

Comment:
Why's this one using a static commit?

Reply:
I don’t know how to build the current version of `Unicoq`. See https://github.com/unicoq/unicoq/pull/17 for details.